### DISCUSSION TITLE
【内容产出挑战】如何写一个基于 Etherscan API 的 Syncer

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#33] [Amazing Things!]
悬赏金额：`¥500`

Description：

在 Gallery 中用 Elixir 实现了一个基于 Etherscan 标准 API 聚焦于 Contract 的 Syncer：

> https://github.com/WeLightProject/tai_shang_nft_gallery

需要产出一篇技术文章，用一门更加普适性的语言实现 Syncer 的逻辑，同时兼顾描述一下 Elixir Syncer 实现的逻辑

**Comment by kenspirit** [comment]
技术文章写在哪里？就在这个 issue 吗？还是根目录下的一个新的 Markdown 文件？

**Reply by leeduckgo** [reply]
> 技术文章写在哪里？就在这个 issue 吗？还是根目录下的一个新的 Markdown 文件？

根目录下新建一个  md 文件，文件名为：
> How to Make a Syncer based on Etherscan API

**Comment by leeduckgo** [comment]
Done
